besu/src/test/java/org/hyperledger/besu/RunnerBuilderTest.java

```diff
@@ -393,4 +399,12 @@ public void noEngineApiNoServiceForMethods() {
     assertThat(runner.getJsonRpcPort()).isPresent();
     assertThat(runner.getEngineJsonRpcPort()).isEmpty();
   }
+
+  private void setupBlockchainAndBlock() {
```

Comment:
can't we just call this method once in the setup()? Just trying to understand what the advantages of having it called in each test instead of in the setup